roles/gui-tools/tasks/setup-regolith.yml

```diff
@@ -51,6 +51,19 @@
   become: true
   when: not regolith_installed.stat.exists
 
+- name: Ensure the /etc/X11/xorg.conf.d directory exists
+  ansible.builtin.file:
+    path: '/etc/X11/xorg.conf.d'
+    state: directory
+  become: true
+
+- name: Append lines to the touchpad.conf file
+  ansible.builtin.blockinfile:
+    path: '/etc/X11/xorg.conf.d/90-touchpad.conf'
```

Comment:
I'm not sure, what happens if this file already exists ? could we totally brick someone's mouse/touchpad access ? :sweat: